lib/MusicBrainz/Server/Edit.pm

```diff
@@ -135,6 +130,14 @@ has 'votes' => (
     }
 );
 
+sub yes_votes {
+    shift->_grep_votes(sub { $_->vote == $VOTE_YES });
+}
+
+sub no_votes {
+    shift->_grep_votes(sub { $_->vote == $VOTE_NO });
+}
```

Comment:
This is the change we talked about on IRC: Just evaluate the votes instead of separately loading vote counts.